lib/viewcomponents/vertex.jsx

```diff
@@ -33,13 +34,22 @@ DIRECTION_TO_LABEL_POSITION[CardinalDirection.WEST] = PortLabelPosition.RIGHT;
 
 var Vertex = React.createClass({
 
+  getInitialState: function() {
+    return {
+      mouseHover: false
```

Comment:
Fixed. I was mistaken about this. There's a similar thing that CSS selectors can't do, but this isn't it.